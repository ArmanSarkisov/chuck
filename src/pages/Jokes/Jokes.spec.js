import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import { Jokes } from './index';

describe('Jokes page test', () => {
  let component;
  beforeEach(() => {
    component = shallow(<Jokes />);
  });

  it('should render Jokes component', () => {
    expect(toJson(component)).toMatchSnapshot();
  });
});
