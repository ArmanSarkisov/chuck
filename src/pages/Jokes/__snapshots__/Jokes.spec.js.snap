// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Jokes page test should render Jokes component 1`] = `
<div
  className="container mx-auto"
>
  <Joke />
  <div
    className="container mx-auto pt-32"
  >
    <Table
      data={Array []}
    />
  </div>
</div>
`;
